uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/leave-request-model_test.js

```diff
@@ -10,8 +10,9 @@ define([
       LeaveRequestAPI,
       $rootScope;
 
-    beforeEach(module('leave-absences.models', 'leave-absences.mocks', function (_$provide_) {
-      $provide = _$provide_;
+    beforeEach(module('leave-absences.models', 'leave-absences.mocks',
```

Comment:
Are these changes coming from your IDE? They don't introduce anything, they just apply some code styling rule. Can you remove them, so we can reduce the noise in this PR?